pkg/urlutil/git.go

```diff
@@ -8,11 +11,13 @@ var (
 		"github.com/",
 		"git@",
 	}
+
+	urlPathWithFragmentSuffix = regexp.MustCompile(".git(:?#.+)?$")
```

Comment:
You sure about that?  When I play with it at: http://play.golang.org/p/X3_7h2Fq7z
I was expecting it to return false for:
```
regexp.MatchString(".git(:?#.+)?$", "foo.git:#abc")
```